tests/devapps/NetCoreTestApp/Program.cs

```diff
@@ -234,8 +234,12 @@ private static IConfidentialClientApplication CreateCca()
         {
             IConfidentialClientApplication cca = ConfidentialClientApplicationBuilder
                 .Create(s_clientIdForConfidentialApp)
+                //.WithAuthority("https://login.microsoftonline.com/12354-12345-123-123-23")
```

Comment:
nit: remove the commented code? if it's needed maybe a comment as to what it can be used for?